api/src/main/java/org/openmrs/module/pihcore/apploader/CustomAppLoaderFactory.java

```diff
@@ -3204,22 +3204,19 @@ private void enableLabs() throws UnsupportedEncodingException {
                 null));
 
         if (config.getCountry() == ConfigDescriptor.Country.SIERRA_LEONE) {
-            apps.add(addToClinicianDashboardFirstColumn(app(CustomAppLoaderConstants.Apps.ALL_LAB_RESULTS,
+            apps.add(addToClinicianDashboardFirstColumn(app(CustomAppLoaderConstants.Apps.RECENT_LAB_RESULTS,
                             "pihcore.lab.lab_results.caps",
                             "fas fa-vial",
                             null,
                             null,
                             objectNode(
-                                    "widget", "latestObsForConceptList",
-                                    "icon", "fas fa-vial",
-                                    "label", "pihcore.lab.lab_results.caps",
-                                    "concepts",
-                                        CustomAppLoaderConstants.SARS_COV2_ANTIBODY_TEST + "," + CustomAppLoaderConstants.SARS_COV2_ANTIGEN_TEST + "," + CustomAppLoaderConstants.SARS_COV2_RT_PCR_TEST + "," + CustomAppLoaderConstants.SARS_COV2_XPERT_TEST + "," + CustomAppLoaderConstants.GLUCOSE_TEST + "," + CustomAppLoaderConstants.BUN_TEST + "," + CustomAppLoaderConstants.NA_TEST + "," + CustomAppLoaderConstants.K_TEST + "," + CustomAppLoaderConstants.CR_TEST + "," + CustomAppLoaderConstants.HB_TEST,
-                                    "conceptNameType", "shortName",
+                                    "encounterTypes", PihEmrConfigConstants.ENCOUNTERTYPE_LAB_SPECIMEN_COLLECTION_UUID + "," + PihEmrConfigConstants.ENCOUNTERTYPE_LAB_RESULTS_UUID,
```

Comment:
It's definitely confusing how we do all this, but I think we want to pull the encounter types from this global property, which is what I believe the Lab Workflow OWA uses...

https://github.com/PIH/openmrs-config-pihemr/blob/master/configuration/globalproperties/gp.xml#L327